projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/sample/AnalysisMethod.java

```diff
@@ -0,0 +1,151 @@
+  WGS("WGS",
+      "Genome sequencing", "Sequencing of the entire genome of an organism"),
+  WES("WES",
+      "Exome sequencing", "Sequencing of all exons of protein-coding genes of an organism"),
+  ATAC_SEQ("ATAC-SEQ", "ATAC sequencing",
+      "Assay for transposase-accessible chromatin with sequencing"),
+  RNA_SEQ("RNA-SEQ", "RNA sequencing", "Detection and quantitative analysis of RNA in a sample"),
+  SC_ATAC_SEQ("SC-ATAC-SEQ", "Single-cell ATAC sequencing",
+      "Assay for transposase-accessible chromatin with sequencing at single-cell resolution"),
+  SC_RNA_SEQ("SC-RNA-SEQ", "Single-cell RNA sequencing",
+      "RNA sequencing at single-cell resolution"),
+  SC_AMPLICON_SEQ("SC-AMPLICON-SEQ", "Single-cell amplicon sequencing",
+      "Amplicon sequencing at single-cell resolution"),
+  ONT_METAGENOMIC("ONT-METAGENOMIC", "Nanopore metagenomics",
+      "Sequencing of the entire genetic content in a community present in a sample with Nanopore technology"),
+  ONT_WGS("ONT-WGS", "Nanopore genome sequencing",
+      "Sequencing of the entire genome of an organism with Nanopore technology"),
+  ONT_RNA("ONT-RNA", "Nanopore RNA sequencing",
+      "Detection and quantitative analysis of RNA in a sample with Nanopore technology"),
+  ONT_APMLICON("ONT-AMPLICON", "Nanopore amplicon sequencing",
+      "Amplicon sequencing with Nanopore technology"),
+  PACBIO_HIFI("PACBIO-HIFI", "PacBio HiFi",
+      "Sequencing of the entire genome of an organism with PacBio technology"),
+  PACBIO_ISOSEQ("PACBIO-ISOSEQ", "PacBio IsoSeq",
+      "Detection and quantitative analysis of RNA in a sample with PacBio technology"),
+  ISOLATION_ONLY("ISOLATION-ONLY", "Isolation only", "DNA and RNA isolation only, no sequencing"),
+  QC_ONLY("QC-ONLY", "QC only", "Quality control only, no sequencing"),
+  SEQ_ONLY("SEQ-ONLY", "Sequencing only", "Processing of ready-to-sequence pools"),
+
+  /*
+  Proteomics related analysis types
+   */
+
+  PROTEOMICS("PROTEOMIICS", "Proteomics", ""),
+
+  PHOSPHO_PROTEOMICS("PHOSPHO", "Phosphoproteomics", ""),
+
+  PEPTIDOMICS("PEPTIDOMICS", "Peptidomics", ""),
+
+  INTERACTORS("INTERACTORS", "Interactors", ""),
+
+  PTMS("PTMS", "Posttransductional mutations", ""),
+
+  /*
+  Metabolomics related analysis types
+   */
+
+  UNTARGETED_MX("UNTARGETED-MX", "Untargeted metabolomics", ""),
+
+  TARGETED_AA("TARGETED-AA", "Targeted amino acids", "" ),
+
+  TARGETED_NUCLEOTIDES("TARGETED-NUCLEOTIDES", "Targeted nucleotides", ""),
+
+  TARGETED_CETO_ACIDS("TARGETED-CETO-ACIDS", "Targeted ceto acids", ""),
+
+  TARGETED_ALL_MX("TARGETED-ALL-MX", "Targeted all metabolites", "");
+
+
+  private static final Map<String, AnalysisMethod> fixedTermToEnum;
+
+  static {
+    fixedTermToEnum = new HashMap<>();
+    for (AnalysisMethod method : AnalysisMethod.values()) {
+      fixedTermToEnum.put(method.term(), method);
+    }
+  }
+
+
+  private final String fixedTerm;
+
+  private final String label;
+
+  private final String description;
```

Comment:
What is a fixed term?

Reply:
platform coordinators use it currently as abbreviations with the labs. Since we have all three properties, we are flexible in the app what to display and how.